### ISSUE TITLE
[Highcharts Accessibility] -- While navigating back (Shift + Tab) focus is not visible on chart if the chart has only 1 data point

### ISSUE THREAD

**Issue by SummerLi** [highcharts/highcharts#8051]
**Specific repro scenario:**

1.	When column chart has multiple data points – not reproduced
-	Using tab will focus on chart columns in first chart container
-	Tab until focus focuses on chart columns in second chart container
-	Shift + Tab, focus will focus on the last chart column in first chart container

2.	When column chart has only one data point – reproduced
-	Using tab will focus on the chart column in first chart container
-	Tab until focus focuses on the chart column in second chart container
-	Shift + Tab, focus will not focus on the chart column in first chart container

#### Affected browser(s)
Chrome

**Comment by SummerLi** [comment]
Repro files: [Demo.zip](https://github.com/highcharts/highcharts/files/1831356/Demo.zip)

After investigating Highcharts accessibility source code, the responsible code is copied below: Since chart only has one data point, the current svgElement will be equal to the last focusElement, so it doesn’t re-draw the focus border.


![code_screenshot](https://user-images.githubusercontent.com/1152425/37684786-5d9052c4-2c4e-11e8-9045-71f64d514c7b.jpg)

**Comment by oysteinmoseng** [comment]
@SummerLi This should be fixed in the latest version of the accessibility module. Could you test with http://github.highcharts.com/modules/accessibility.js?

**Comment by KacperMadej** [comment]
Closing due to inactivity. The issue could be reopened when more information will be provided.
